<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QMainWindow" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	font-size:20px;
	font-weight;bold;
	font-family:微软雅黑;
}
QWidget{
	background-color:rgb(97,111,125);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
background-color:white;
}</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="MyWeatherWidget" name="widgetToday" native="true"/>
       </item>
       <item>
        <widget class="MyWeatherWidget" name="widgetFuture1" native="true"/>
       </item>
       <item>
        <widget class="MyWeatherWidget" name="widgetFuture2" native="true"/>
       </item>
       <item>
        <widget class="MyWeatherWidget" name="widgetFuture3" native="true"/>
       </item>
       <item>
        <widget class="MyWeatherWidget" name="widgetFuture4" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QLabel" name="labelTime">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>更新时间:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="exitBtn">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>40</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img/res.qrc">
         <normaloff>:/back.png</normaloff>:/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWeatherWidget</class>
   <extends>QWidget</extends>
   <header>myweatherwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img/res.qrc"/>
  <include location="img/res.qrc"/>
 </resources>
 <connections/>
</ui>
